lighthouse-core/config/constants.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
```

Comment:
might be dumb, but: maybe `defaults.js` or something? It might be good to have it be specifically a place where we keep defaults for these kinds of controls and avoid the temptation of just having it be a grab bag of constants we need across files

Reply:
point taken but can we please pick another name? `default.js` *and* `defaults.js` would just be nuts 😆 maybe we rename `default.js -> default-config.js`?

Reply:
yeah agreed, let's do this in followup, how about I just leave the Driver require for now and address all these in separate go for review?

Reply:
SGTM